### PR TITLE
Add mborsz to perfdash/OWNERS

### PR DESCRIPTION
**Pull request by mborsz**
Also reorders approvers to match reviewers (and to match lexicographic order)
/assign @wojtek-t

### CODE REVIEW

File: perfdash/OWNERS [hunk 1]
```diff
@@ -1,8 +1,9 @@
 approvers:
 - krzysied
+- mborsz
+- oxddr
```
Comment (wojtek-t) [inline perfdash/OWNERS:4]: Let's remove Janek from that list.

Comment (mborsz) [reply perfdash/OWNERS]: DONE

**Comment by mborsz** [comment]
PTAL

**Comment by wojtek-t** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**